6-AdvancedScenarios/4-hybrid-SPA/README.md

```diff
@@ -0,0 +1,412 @@
+---
+page_type: sample
+name: Hybrid Express web app and React SPA sample
+services: active-directory
+platforms: React & Express
+urlFragment: ms-identity-javascript-react-tutorial 
+description: This sample demonstrates an Express web app calling a React SPA using the Hybrid SPA flow
+languages:
+ - JavaScript
+products:
+ - React
+ - Express
+---
+
+# Sign-in users interactively server-side (Node.js) and silently acquire token for MS Graph for a React Single-page app (SPA)
+
+[![Build status](https://identitydivision.visualstudio.com/IDDP/_apis/build/status/AAD%20Samples/.NET%20client%20samples/ASP.NET%20Core%20Web%20App%20tutorial)](https://identitydivision.visualstudio.com/IDDP/_build/latest?definitionId=819)
+
+Table Of Contents
+
+* [Scenario](#Scenario)
+* [Prerequisites](#Prerequisites)
+* [Setup the sample](#Setup-the-sample)
+* [About the code](#about-the-code)
+* [Troubleshooting](#Troubleshooting)
+* [Next Steps](#Next-Steps)
+* [Contributing](#Contributing)
+* [Learn More](#Learn-More)
+
+## Scenario
+
+This sample demonstrates how a React SPA app can silently redeem an authorization code for an [access token](https://aka.ms/access-tokens) for **Microsoft Graph API** for a user who has already authenticated earlier in another Node.Js application
+
+The Express web application will interactively authenticate a user using **MSAL-Node** and obtains both an [Access Token](https://aka.ms/access-tokens) and as well as an additional [Spa Authorization Code](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/wiki/SPA-Authorization-Code) using the Hybrid SPA flow.
```

Comment:
>https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/wiki/SPA-Authorization-Code [](http://example.com/codeflow?start=206&length=98)

Is there a MSAL Node or MSAL.js page on this topic?

Reply:
not at the moment, no